src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.tsx

```diff
@@ -26,30 +28,102 @@ const Loader = () => (
   </div>
 )
 
+const BUNDLER_OPTIONS = {
+  Vite: 'Vite',
+  Rollup: 'Rollup',
+  Webpack: 'Webpack',
+} as const
+type BundlerOption = keyof typeof BUNDLER_OPTIONS
+type BundlerOptionUrls = Record<BundlerOption, string>
+
+const getInitialBundler = (path: string, urls: BundlerOptionUrls) => {
+  if (path === urls.Vite) {
```

Comment:
Couldn't come up with a type safe way to do this with one generic case. LMK if you can think of something 😄